update.sh

```diff
@@ -9,13 +9,21 @@ if [ ${#versions[@]} -eq 0 ]; then
 fi
 versions=( "${versions[@]%/}" )
 
+# https://www.rabbitmq.com/which-erlang.html ("Maximum supported Erlang/OTP")
+declare -A otpMajors=(
+	[3.7]='21'
+)
+
+# https://www.openssl.org/policies/releasestrat.html
+# https://www.openssl.org/source/
+declare -A opensslMajors=(
+	[3.7]='1.1'
+)
```

Comment:
@michaelklishin @gerhard do you think this is the right level of granularity on these two versions for upgrading?

From https://www.rabbitmq.com/which-erlang.html, I see that 3.7 should max out at 21.x (which is why I put that granularity there, easily adjusted to be more granular like `21.2`, for example).

On OpenSSL, I feel less sure.  Reading https://www.openssl.org/policies/releasestrat.html, it seems any 1.1.x should be binary compatible, which doesn't really matter here, but seemed like a good differentiator to make sure we don't get broken by something changing in 1.2.x.

Reply:
OTP support is no longer tied to a version, it is tied to a time frame (2 major most recent major versions, up to 3 for a transition period). So 3.7.x almost certainly will support OTP 22 (unless there are breaking changes that are catastrophic for us) but OTP 22 won't ship till June.

I agree with @gerhard, for now assuming that OTP `21.x` will imply/require OpenSSL `1.1.x` is a pretty safe bet. That said, the rate of change and refactoring in Erlang's `crypto` and `ssl` modules has been very high in the last ~ 6-7 months due to TLSv1.3 work and general long overdue house cleaning. But I doubt TLSv1.3 is coming to OTP `21.x`.